src/applications/appeals/995/components/ContactInfo.jsx

```diff
@@ -0,0 +1,226 @@
+import React, { useEffect, useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+import AddressView from '@@vap-svc/components/AddressField/AddressView';
+
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+import { scrollAndFocus } from 'platform/utilities/ui';
+
+import { readableList } from '../utils/helpers';
+import { getFormatedPhone } from '../utils/contactInfo';
+import { content } from '../content/contactInfo';
+
+const ContactInfo = ({
+  data,
+  goBack,
+  goForward,
+  onReviewPage,
+  updatePage,
+  contentBeforeButtons,
+  contentAfterButtons,
+}) => {
+  const [hadError, setHadError] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+  const wrapRef = useRef(null);
+  window.sessionStorage.setItem('onReviewPage', onReviewPage || false);
```

Comment:
I'm guessing that this doesn't affect the way it works, but won't this get set every time the component re-renders?

Reply:
Probably. Maybe it needs to be inside a `useEffect`?

Reply:
I added a console log and it appears to get called infrequently. The content of the page is mostly static.